intervention_compassion/models/compassion_intervention.py

```diff
@@ -80,20 +82,69 @@ class CompassionIntervention(models.Model):
         ("GP Preferences Submitted", _("GP Preferences Submitted")),
         ("GP Rejected Costs", _("GP Rejected Costs")),
     ])
+    # TODO USD or not USD?
```

Comment:
We use USD but don't specify in the name fields. We just ignore other currencies.